src/wrapper.rs

```diff
@@ -209,7 +218,7 @@ where
         buf,
         "fn default_instance() -> &'static {}{} {{
             ::lazy_static::lazy_static! {{
-                static ref INSTANCE: {0}{1} = {0}{1}::default();
+                static ref INSTANCE: {0}{1} = {0}{1}::new();
```

Comment:
We have a generated `new` which already uses what we want (so no need another `if` here)

Reply:
I think you might need to use fully explicit syntax because there is an inherent `new` (I don't recall if there are shadowing rules). Have you tested using this codegen in kvproto?

Reply:
> I think you might need to use fully explicit syntax because there is an inherent `new`

Ok.

> Have you tested using this codegen in kvproto?

Not yet, I can do it. Which branch do you recommend me to get started with?

Reply:
Done